corehq/apps/oauth_integrations/models.py

```diff
@@ -14,3 +15,30 @@ class LiveGoogleSheetSchedule(models.Model):
     is_active = models.BooleanField(default=True)
     start_time = models.IntegerField(default=200)
     google_sheet_id = models.CharField(max_length=250)
+
+
+class LiveGoogleSheetErrorReason():
+    NO_ERROR = None
+    INVALID_TOKEN = 'token'
+    TIMEOUT = 'timeout'
+    OTHER = 'other'
+
+    CHOICES = (
+        (NO_ERROR, "No Error"),
+        (INVALID_TOKEN, "Invalid Token"),
+        (TIMEOUT, "Data Timeout"),
+        (OTHER, "Other..."),
+    )
+
+
+class LiveGoogleSheetRefreshStatus(models.Model):
+    schedule = models.ForeignKey(LiveGoogleSheetSchedule, on_delete=CASCADE)
```

Comment:
This value of `CASCADE` is not valid. This is `tkinter.CASCADE` when it should be `django.db.models.CASCADE`. I don't know how these migrations are successfully applied in Django 2.2, but it [causes tests to fail in Django 3.2](https://github.com/dimagi/commcare-hq/runs/5571257166?check_suite_focus=true).

In addition to fixing this, maybe also consider adding a test to ensure that cascading deletes perform as expected.